packages/core/lib/search/metaheuristics/SearchAlgorithm.ts

```diff
@@ -112,17 +111,17 @@ export abstract class SearchAlgorithm<T extends Encoding> {
 
     // Start initialization budget tracking
     budgetManager.initializationStarted();
-    getUserInterface().startProgressBar();
+    // getUserInterface().startProgressBar();
```

Comment:
What is the plan for these types of information?

Reply:
We can use the event listener for this.

Reply:
This is actually being fixed in the user-interface refactor in another branch/PR

Reply:
Then we should probably remove this code entirely in this PR

Reply:
You mean the commented code?
I want to leave it in such that I remember where to put the new function calls.